ci/setup_grpc.sh

```diff
@@ -144,8 +146,8 @@ echo "Building gRPC ${install_grpc_version}"
 echo "CMake build options:" "${GRPC_BUILD_OPTIONS[@]}"
 
 cmake "${GRPC_BUILD_OPTIONS[@]}" ..
-cmake --build . -j$(nproc)
-cmake --install .
+make -j $(nproc)
```

Comment:
Why use `make -j` here? From my understanding, `cmake --build . -j` offers better compatibility, supporting Ninja and other generators.

`cmake --install .` also offers better compatibility than `make install`.